server/templates/staff/student/submit.html

```diff
@@ -43,7 +43,7 @@
           </p>
 
           {% call forms.render_form(upload_form, class_='form', enctype="multipart/form-data") %}
-            {{ forms.render_field(upload_form.upload_files, label_visible=true, multiple="", id="file-select") }}
+            {{ forms.render_field(upload_form.upload_files, label_visible=true, multiple="", id="file-select", webkitdirectory='true') }}
```

Comment:
Browser support for this is limited since it's still a draft spec: 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/webkitdirectory


IE Edge - Yes. 
Safari (which uses webkit...) - No